apps/coordinator/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "rootDir": "src",
```

Comment:
I added this at one point when trying to debug various compiler and jest/testing issues resulting from the new libraries and some newer esmodule dependencies. Not sure if this was necessary, but didn't hurt. 

Long term I'd like to improve the reusability of our dev tool configs. It's all a bit fragile because of the legacy code and I didn't want to risk breaking anything when everything was moved to monorepo.